app/exporters/patients_with_history_exporter.rb

```diff
@@ -29,6 +29,17 @@ class PatientsWithHistoryExporter
 
   attr_reader :display_blood_sugars
 
+  def self.json(*args)
+    new.json(*args)
+  end
+
+  def json(patients, display_blood_sugars: true)
+    @display_blood_sugars = display_blood_sugars
+    summaries = MaterializedPatientSummary.where(patient: patients)
+
+    summaries.to_json
```

Comment:
We will need to get fine-grained with the JSON presentation of a patient summary. Matching the CSV list would be a good first step.